### ISSUE TITLE
Improve the appearance of morphed expanded subprocesses

### ISSUE THREAD

**Issue by Skaiir** [bpmn-io/bpmn-js#1632]
__Is your feature request related to a problem? Please describe.__

Currently when morphing to an expanded subprocess, we do not resize the bounds. This is confusing because it is visually not different to a Task (https://github.com/camunda/camunda-modeler/issues/1845).

__Describe the solution you'd like__

Morphing should expand the bounds to give a visual indication that we're dealing with a sub-process and not a task. I'm not certain if we should just match the default size but at the very least expand it a little bit.

__Describe alternatives you've considered__

Give expanded sub-processes a visually distinctive feature to differentiate them to tasks.

**Comment by smbea** [comment]
We resize the expanded subprocess to be like so:
<img width="458" alt="Screenshot 2022-04-22 at 09 26 54" src="https://user-images.githubusercontent.com/25825387/164627424-b7f13ceb-6a7d-44fd-85ae-964d859fdef0.png">

Do you mean to make it larger than this?

**Comment by Skaiir** [comment]
@smbea Yeah the idea would be to scale it at least to a minimum that we're happy with. Something that makes sense vertically and horizontally. I think the behavior should be resize to min_width * min_height, and then let automatic resizing based on children happen.

**Comment by smbea** [comment]
Ok then I will keep this open
